moveit_ros/moveit_servo/include/moveit_servo/servo_parameters.h

```diff
@@ -85,6 +85,8 @@ struct ServoParameters
   std::string move_group_name;
   std::string planning_frame;
   std::string ee_frame_name;
+  bool is_primary_planning_scene_monitor;
```

Comment:
Should we add this to the sample servo config files in moveit2 repo and moveit2_tutorials repo?